web/src/testgrid-tab-table.ts

```diff
@@ -0,0 +1,61 @@
+import { LitElement, html, css } from 'lit';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { customElement, property } from 'lit/decorators.js';
+import {TabSummaryInfo} from './testgrid-dashboard-summary';
+
+@customElement('testgrid-tab-table')
+export class TestgridTabTable extends LitElement {
+  @property() clicked = '';
+  @property() visible = false;
```

Comment:
this should probably be a state and not a property. please refer to https://lit.dev/docs/components/properties/#internal-reactive-state

Reply:
why shouldn't we use a property for this?

Reply:
property is an externally set attribute while state is internal. looks like visible is toggled within the table element

Reply:
fixed